notifications/src/main/java/dk/shape/games/notifications/presentation/viewmodels/notifications/SubjectNotificationSwitcherViewModel.kt

```diff
@@ -25,6 +28,12 @@ internal class SubjectNotificationSwitcherViewModel {
 
     val item: ObservableField<Any> = ObservableField(loadingViewModel)
 
+    init {
+        item.onChange {
+            onItemChanged()
+        }
+    }
+
```

Comment:
maybe we can just call `onChange`directly on the item to avoid the init block?
```suggestion
     val item: ObservableField<Any> = ObservableField<Any>(loadingViewModel).onChange {
        onItemChanged()
    }
```

Reply:
I've tried, but then I get an error because we can't initialize it with loadingViewmodel anymore since it's expecting a view model of that type

Reply:
You just need to specify the type `ObservableField<Any>`